src/components/controls/LanguageSelector.vue

```diff
@@ -21,11 +21,10 @@ export default {
     },
 
     options() {
-      let options = {};
-      this.languageList.forEach(lang => 
-        { options[lang] = lang }
-      );
-      return options;
+      let options = {}
+      this.languageList.forEach(lang => { options[lang] = lang }
+      )
```

Comment:
This parenthesis looks weird :)

Reply:
it's auto fix by linter, let me try add some rules on linter